doc/api/esm.md

```diff
@@ -231,6 +231,28 @@ and [absolute specifiers][Terminology]. Resolving
 from `data:text/javascript,import "./foo";` fails to resolve because there
 is no concept of relative resolution for `data:` URLs.
 
+#### `blob:` imports
+
+<!-- YAML
+added: REPLACEME
+-->
+
+`blob:` URLs are supported for importing with the following MIME types:
+
+* `text/javascript` for ES modules
+* `application/json` for JSON
+* `application/wasm` for Wasm
+
+```mjs
+import { Blob } from 'node:buffer';
+
+const json = new Blob(['{}'], { type: 'application/json' });
```

Comment:
The documentation is stating support for one mimetype, `text/javascript`, but the example is showing a different one, `application/json`. I recognize they are equivalent, but the docs should be clear.

Reply:
According to the documentation:

> `blob:` URLs are supported for importing with the following MIME types:
>
> * `text/javascript` for ES modules
> * `application/json` for JSON
> * `application/wasm` for Wasm

Am I missing something?

Reply:
The string `text/javascript` does not also include the string `application/javascript`.

Reply:
While `application/javascript` is a valid mime type, it isn't documented for `data:` urls, so I didn't document it here.

Reply:
Then what is the purpose of the example using `application/json`?

Reply:
To show JSON support, which is also documented

Reply:
You should probably re-order the examples so it appears in the same order as the list above.

Reply:
Oh wow. Sorry folks. My eyes must not be connected to my brain this morning. I was reading and writing very different things. Ignore this whole thread.

Reply:
No problem! That happens to me a whole lot more than I'd like to admit.